tests/test_scheduler.py

```diff
@@ -667,12 +666,10 @@ def test_steps_offset(self):
         scheduler_config = self.get_scheduler_config(steps_offset=1)
         scheduler = scheduler_class(**scheduler_config)
         scheduler.set_timesteps(10)
-        assert torch.equal(
+        assert np.equal(
             scheduler.timesteps,
-            torch.tensor(
-                [901, 851, 851, 801, 801, 751, 751, 701, 701, 651, 651, 601, 601, 501, 401, 301, 201, 101, 1]
-            ),
-        )
+            np.array([901, 851, 851, 801, 801, 751, 751, 701, 701, 651, 651, 601, 601, 501, 401, 301, 201, 101, 1]),
```

Comment:
Same comment as above. I think this needs to be reverted if we are moving to pytorch-only code.

Reply:
Since `timesteps` need to be in NumPy format, I assume this is necessary. So I'll close this comment.